src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_osa_commands.py

```diff
@@ -153,3 +153,37 @@ def test_openshift_create_default_service_no_aad(self, resource_group, resource_
 
         # delete
         self.cmd('openshift delete -g {resource_group} -n {name} --yes --no-wait', checks=[self.is_empty()])
+    # It works in --live mode but fails in replay mode.get rid off @live_only attribute once this resolved
+    @live_only()
+    @ResourceGroupPreparer(random_name_length=17, name_prefix='clitestosa', location='eastus')
+    @ManagedApplicationPreparer()
+    def test_openshift_create_without_monitoring(self, resource_group, resource_group_location, aad_client_app_id, aad_client_app_secret):
+        # kwargs for string formatting
+        osa_name = self.create_random_name('clitestosa', 15)
+        self.kwargs.update({
+            'resource_group': resource_group,
+            'name': osa_name,
+            'location': resource_group_location,
+            'aad_client_app_id': aad_client_app_id,
+            'aad_client_app_secret': aad_client_app_secret
+        })
+
+        # create
+        create_cmd = 'openshift create --resource-group={resource_group} --name={name} --location={location} ' \
```

Comment:
is it able to link a workspace since this PR is related with `workspace`? We have `az monitor log-analytics workspace create`